unlock-app/src/components/interface/checkout/main/Minting.tsx

```diff
@@ -48,8 +73,7 @@ export function Minting({
 
           const transaction = await provider.waitForTransaction(
             mint!.transactionHash!,
-            2,
-            60000
+            2
```

Comment:
Removed the timeout here... I am not sure what it adds and it may be misleading because when the timeout occurs it does not mean that the transaction will not eventually be mined.

Reply:
Just to stop a situation where the transaction is taking too long and show loading.